runpod/serverless/worker.py

```diff
@@ -49,53 +49,77 @@ def _is_local(config) -> bool:
 # ------------------------- Main Worker Running Loop ------------------------- #
 async def run_worker(config: Dict[str, Any]) -> None:
     """
-    Starts the worker loop.
+    Starts the worker loop for multi-processing.
+
+    Args:
+        config (Dict[str, Any]): Configuration parameters for the worker.
     """
     auth_header = _get_auth_header()
-    async with aiohttp.ClientSession(headers=auth_header, timeout=_TIMEOUT) as session:
+    connector = aiohttp.TCPConnector(limit=None)
```

Comment:
I wonder if we can control the number of jobs being requested by simply adjusting this parameter instead. 

Not something required now, but can keep it in mind.

Reply:
Great question. One thing to be careful about is that the rate needs to adjust over time due to changes in workload distribution. But yes, creating a simpler parameterization of the concurrency would be very beneficial in improving maintainability.